grails-app/controllers/org/pih/warehouse/picklist/PicklistController.groovy

```diff
@@ -82,4 +89,83 @@ class PicklistController {
         println location
         render(template: "/picklist/print", model: [requisition: requisition, picklist: picklist, location: location, order: params.order])
     }
+
+    def exportPicklistItems() {
+        String format = params.get("format", "csv")
+
+        List<PickPageItem> pickPageItems = stockMovementService.getPickPageItems(params.id, null, null )
+        List<PicklistItem> picklistItems = pickPageItems.picklistItems?.flatten()
+
+        List lineItems = picklistItems.collect {
+            [
+                    id: it?.requisitionItem?.id,
+                    code: it?.requisitionItem?.product?.productCode,
+                    name: it?.requisitionItem?.product?.name,
+                    lot: it?.inventoryItem?.lotNumber,
+                    expiration: it.inventoryItem.expirationDate?.format(Constants.EXPIRATION_DATE_FORMAT),
+                    binLocation: it?.binLocation?.name,
+                    quantity: it?.quantity,
+            ]
+        }
+
+        String fileName = "PickListItems\$-${params.id}"
+
+        switch (format) {
+            case "csv":
+                OutputStream outputStream = new ByteArrayOutputStream()
+                DataExporter pickListItemCsvExporter = new PicklistItemCsvExporter(lineItems)
```

Comment:
Looking into Justin's comments, I thought it was meant to be moved and designed in OBIG, and here we wanted to use methods from the `dataService`. Let me know if I missed some discussion around it.